packages/flutter_tools/lib/src/windows/visual_studio_validator.dart

```diff
@@ -47,6 +47,18 @@ class VisualStudioValidator extends DoctorValidator {
         messages.add(ValidationMessage(_userMessages.windows10SdkVersion(windows10SdkVersion)));
       }
 
+      if (_visualStudio.cmakePath != null) {
+        final String? cmakeVersion = _visualStudio.getCmakeVersion();
+        if (cmakeVersion != null) {
+          if (double.parse(cmakeVersion) >= 3.15) {
+            messages.add(ValidationMessage(_userMessages.visualStudioCmakeVersion(cmakeVersion)));
```

Comment:
The CMake version is a function of the VS version; it's not independently controlled. Given that, I don't see any value (either to developers, or to us) in reporting it here.

Reply:
I added this based on https://github.com/flutter/flutter/issues/88589#issuecomment-903098168. We don't to provide a CMake validation message

Reply:
> I added this based on [#88589 (comment)](https://github.com/flutter/flutter/issues/88589#issuecomment-903098168).

That comment seems to be asking for a minimum VS version check (i.e., my preferred solution, as described in my comment above.)

> We don't provide a CMake validation message?

Sorry, I'm not sure what the question is.